_maps/map_files/Reno/Reno-Surface-2.dmm

```diff
@@ -231,8 +231,8 @@
 /area/f13/building)
 "bX" = (
 /obj/structure/shelf_wood,
-/obj/item/gun/ballistic/rifle/hobo/plasmacaster,
```

Comment:
Are plasmacasters this bad?

Reply:
I don't think it's a balanced weapon to start with. If they had to buy it, it would be fine.